src/ranged.cpp

```diff
@@ -693,7 +702,7 @@ dealt_projectile_attack player::throw_item( const tripoint &target, const item &
 // Draws the static portions of the targeting menu,
 // returns the number of lines used to draw instructions.
 static int draw_targeting_window( WINDOW *w_target, item *relevant, player &p, target_mode mode,
-                                  input_context &ctxt )
+                                  input_context &ctxt, std::vector<aim_type> *aim_types )
```

Comment:
Why did you use a pointer and not a reference here? This apparently can never be null, and if it could be null, why is it not checked in the code that uses it?

Also, it should probably be a pointer/reference to a const object.